setup.py

```diff
@@ -61,12 +61,12 @@
 
 _onnxruntime_deps = ["onnxruntime>=1.0.0"]
 _pytorch_deps = [
-    "torch>=1.1.0,<1.9.2",
+    "torch>=1.1.0,<=1.12.1,!=1.10.*,!=1.11.*",
```

Comment:
for torch 1.10 and 1.11, we should avoid using != for requirements since it may cause issues with existing environments, and force upgrades.
instead, let's 

* add a reasonable place to fail early with an exception
* offer an environment variable to override
* if the environment variable is enabled, but they try to export a quantized model, raise a warning

Reply:
Adding in #1088

Reply:
Merged into this branch